templates/spec/solidus_starter_frontend_spec_helper.rb

```diff
@@ -85,6 +85,7 @@ module SolidusStarterFrontend; end
 
     example.run
 
+    Rails.cache.clear
```

Comment:
This seems to be doing a slightly different thing than the original, probably not an issue, unless it will show up only for some execution sequences.

For 100% compatibility I suggest moving it to a `config.before { Rails.cache.clear }`.

Reply:
Or we can move it above the `example.run` line, as we're within an `around(:each)` block.

Reply:
@waiting-for-dev that would probably work as well, although this `around` block is targeting examples with metadata containing `caching: true`.

Reply:
> that would probably work as well, although this around block is targeting examples with metadata containing caching: true.

Right, but if the cache is disabled for other specs, does it matter? I think that line where it was before is a bit like shooting a mosquito with a bazooka and we can do better.